resources/guest_configs/microvm-kernel-arm64-5.10-no-sve.config

```diff
@@ -2892,7 +2892,11 @@ CONFIG_HAVE_FTRACE_MCOUNT_RECORD=y
 CONFIG_HAVE_SYSCALL_TRACEPOINTS=y
 CONFIG_HAVE_C_RECORDMCOUNT=y
 CONFIG_TRACING_SUPPORT=y
-# CONFIG_FTRACE is not set
+CONFIG_FTRACE=y
```

Comment:
Any idea why this matters?

Reply:
Not really. From what I know about ftrace is that it adds some `nop` instructions at the start of each function and those `nop` bytes can in the future be changed to represent a call to the tracer. I guess these `nop` instructions give time buffer for something to happend (initialize or smth) and because of this everything works. I can investigate further, but the amout of work in respect to usefulness of that work is questionable.

Reply:
Fair enough. How did you end up disabling it?

Reply:
Disabling what? Ftrace? It is disabled by default i.e. the bytes are set `nop` at boot time.

Reply:
I'm just trying to reconstruct how you first arrived at the config without ftrace (what was your default config) and why you decided to enable it.

Reply:
The default was our default 5.10 config. This ftrace hack I found when I was investigation this issue in the first place (bach when I was gathering baselines for c7g)

Reply:
Right, so we enable ftrace not because it was enabled in some other config that we use, but empirically, because it just solves the boot problem?

Reply:
Not because it slows the boot process (even thought it does), but because it solves the problem. Most likely by slowing boot, but I am not a grey beard kernel engineer to tell for sure, not yet at least.

Reply:
(I didn't say anything about slowing the boot process, btw)
By all my questions I tried to gauge the dirtiness of this hack. It is certainly fine to apply it if it serves the purpose. I would only suggest adding this in the commit message for future reference. Maybe something in line with "ftrace is a completely random thing, but for some reason it does something that makes the kernel boot successfully. We don't know what specifically, but that's what we need, and we can't afford investigate further at this point".

Reply:
Sorry in my mind "solves the boot problem" became "slows the boot process"...
I will add this comment to commit message.